src/GraphQL/Conversion/PascalCaseFieldNameConverter.cs

```diff
@@ -0,0 +1,17 @@
+using GraphQL.Introspection;
+using System;
+using System.Linq;
+
+namespace GraphQL.Conversion
+{
+    public class PascalCaseFieldNameConverter : IFieldNameConverter
+    {
+        private static readonly Type[] IntrospectionTypes = { typeof(SchemaIntrospection) };
+
+        public static readonly PascalCaseFieldNameConverter Instance = new PascalCaseFieldNameConverter();
```

Comment:
should `instance` be public?

Reply:
Well this is a common approach - make such static readonly instance. Otherwise, the meaning of reuse by the calling code is lost.

Reply:
yeah, just my default approach when any api is added to the public surface area.

Reply:
Ok. Note that for the verification of the public api changes we have #1247 .